lib/kennel/syncer.rb

```diff
@@ -182,5 +178,18 @@ def tracking_id(a)
     def tracking_field(a)
       a[:message] ? :message : :description
     end
+
+    def filtered_resources_from_dd
+      Progress.progress "Downloading definitions" do
+        download_definitions.select do |a|
```

Comment:
maybe a `.reject` to not need the else ?

Reply:
actually looks kind of worse:
```
      download_definitions.reject do |a|
        tracking_id = tracking_id(a)
        !tracking_id.nil? && !tracking_id.start_with?("#{@project_filter}:") if @project_filter
      end
```